### DISCUSSION TITLE
中保科前進 CES 2023 大展！迎接 AIoT 趨勢將開發「虛擬警衛系統」

### DISCUSSION THREAD

**Discussion by Yilin20021016** [1101-GS3073-C/Python-Community-GS3073-C#670] [General]
https://finance.technews.tw/2023/01/06/virtual-guard/
好奇如果出來後被黑點會怎樣

**Comment by HsuChen88** [comment]
我認為警衛的人像應該要找令人畏懼的樣貌，以達到警示的效果!

**Comment by bww0401** [comment]
透過AI確實能夠解決人力成本的問題，但是我認為區區AI應該沒辦法達到嚇阻的作用，該省的還是不能省

**Comment by skyling0299** [comment]
此套虛擬安全系統的主要功能有 3 個：「安全監控」、「接待」和「緊急回應」。透過內建的鏡頭和人臉辨識功能，AI 可確定訪客的特性和目的並適當回應。虛擬警衛也能透過語音辨識和語音合成技術，用自然語言回答訪客的問題，例如，他們會蹲下來和孩子視線一樣高來回答孩童的問題，發現可疑的人也會提出警告。

**Comment by 0212Jerry** [comment]
虛擬警衛的確可以節省人力成本，不過也隱含危險，若沒有相關警急事件的應對系統將會對整個大樓或是商場帶來無法進出或逃生等極大的危險。
